doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2775,14 +2775,35 @@ timestamps.
 :FilePIDsEnabled
 ++++++++++++++++
 
-Toggles publishing of file-level PIDs for the entire installation. By default this setting is absent and Dataverse Software assumes it to be true. If enabled, the registration will be performed asynchronously (in the background) during publishing of a dataset.
+Toggles publishing of file-level PIDs for the entire installation. By default this setting is absent and Dataverse Software assumes it to be false. If enabled, the registration will be performed asynchronously (in the background) during publishing of a dataset.
```

Comment:
By default this setting is no longer absent, right?

Reply:
No - a new installation will not have it set, which we now interpret as false. Existing installs which had it not set will now have it set true.